tools/CMakeLists.txt

```diff
@@ -1,8 +1,12 @@
 add_subdirectory(replay)
-add_subdirectory(toascii)
 add_subdirectory(compress)
 add_subdirectory(info)
 add_subdirectory(extract)
 add_subdirectory(optimize)
 add_subdirectory(capture)
 add_subdirectory(gfxrecon)
+
+# TOASCII build is currently disabled pending resolution of some issues
+if(BUILD_TOASCII)
```

Comment:
I'm mildly concerned that this will impact developers.  I agree that this is clean, but the SDK builds don't currently require this; so it's debatable whether a change that helps no one and inconveniences some people is worth it...